src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,66 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class HomePage extends BasePage {
+    String newPlaylistName1 = "VIVO";
+
+    By avatarIcon = By.cssSelector("[[class='avatar']]");
+    By newPlaylist = By.cssSelector("[data-testid='playlist-context-menu-create-simple']");
+    By clickPlusButton1 = By.cssSelector("[data-testid='sidebar-create-playlist-btn']");
+    By playlistNameField = By.cssSelector("[name='name']");
+    By notificationMsg = By.cssSelector("div.success.show");
+    By playlist = By.xpath("//section[@id='playlists']//li//a[contains(text(),'vivo')]");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```